common/cpp/src/google_smart_card_common/pp_var_utils/construction.h

```diff
@@ -85,6 +85,9 @@ inline pp::Var MakeVar(const std::vector<T>& value) {
   return result;
 }
 
+// Creates an array buffer.
```

Comment:
TODO: Add code on the JS side that would convert ArrayBuffer into Uint8Array when sending cross-extension messages. (Chrome doesn't support ArrayBuffer in cross-extension messages, unfortunately.)

Before this is done, the CL shouldn't be committed, since it'll be a regression.

Reply:
This is done in #226. After that one lands, will land the current PR as well.